systems/controllers/endeffector_velocity_controller.cc

```diff
@@ -76,17 +78,16 @@ void EndEffectorVelocityController::CalcOutputTorques(
   VectorXd commandedTorques(num_joints_);
   commandedTorques = frameSpatialVelocityJacobian.transpose() * generalizedForces;
 
-  // Limit maximum commanded torques
-  double max_torque_limit = 3.0;
+  // Limit maximum commanded velocities
   for (int i = 0; i < num_joints_; i++) {
-      if (commandedTorques(i, 0) > max_torque_limit) {
-          commandedTorques(i, 0) = max_torque_limit;
+      if (commandedTorques(i, 0) > joint_torque_limit_) {
```

Comment:
Since `commandedTorques` is a `VectorXd`, not a `MatrixXd`, we can use `commandedTorques(i)` rather that `(i,0)`.